spec/lib/sm/client/session_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sm/client'
+
+RSpec.describe 'sm client', type: :request do
+  let(:user_id) { '10616687' }
+  let(:current_user) { build(:user, :mhv) }
+
+  before do
+    sign_in_as(current_user)
+    allow(User).to receive(:find).and_return(current_user)
+  end
+
+  def toggle_feature_flag_for_actor(value)
+    allow(Flipper).to receive(:enabled?).with(:mhv_secure_messaging_cerner_pilot, current_user).and_return(value)
```

Comment:
allow(Flipper).to receive(:enabled?).with(: mhv_secure_messaging_cerner_pilot, instance_of(User)).and_return(true)